iOS/Flows/Conversation/Input/MessageDeliveryTypeBadgeView.swift

```diff
@@ -15,52 +15,84 @@ class MessageDeliveryTypeBadgeView: BaseView {
     private var cancellables = Set<AnyCancellable>()
 
     private let imageView = UIImageView()
+    private let label = ThemeLabel(font: .small)
+    private let deliveryTypeLabel = ThemeLabel(font: .xtraSmall)
```

Comment:
This now animates to show a title and description making it much less ambiguous.